test/acceptance/committee_election_test.go

```diff
@@ -25,6 +26,13 @@ func TestCommitteeElections_OneElectionAndCheckReputationChanges(t *testing.T) {
 		WithManagementPollingInterval(20*time.Millisecond).
 		WithLogFilters(log.DiscardAll()).
 		WithConsensusAlgos(consensus.CONSENSUS_ALGO_TYPE_LEAN_HELIX).
+		WithConfigOverride(func(cfg config.OverridableConfig) config.OverridableConfig {
```

Comment:
are you sure we need this here ?
did this cause you flakiness ? maybe we can change the acceptance test grace in the services_config to be 0 instead of this repeated code ? (https://github.com/orbs-network/orbs-network-go/blob/27ac10057b27e8f3ebe30d2abdc6ba1b106313e6/config/system_factory_presets.go#L198)